docs/vision.md

```diff
@@ -0,0 +1,33 @@
+# Our Vision for an Interoperable Transit Industry
+
+More than ever, meeting the needs of travelers requires the complete interoperability of mobility data used by transit providers and transportation technology companies.
+
+**All people** can, equitably:  
+
+- Discover and make informed choices about their mobility options in scheduled and/or real-time by utilizing the highest possible quality of information with the least possible barriers to accessing that information;  
+- Maintain their privacy and security; and  
+- Travel with the minimum possible friction arising from accessing data related to their mode(s) of travel.  
+
+**[Mobility service providers](definitions.md#mobility_service_provider)** can:  
+
+- Interface with other mobility providers to meet riders’ needs;  
+- Use the best available data to support analytics and real-time decision-making for their planning, scheduling, and operating needs;  
+- Select the best available technology and equipment to provide their service without vendor lock-in; and  
+- Easily use their data and data from other [mobility service providers](definitions.md#mobility_service_provider) to increase rider satisfaction and the customer experience through service efficiency and service quality.  
```

Comment:
sounds like we're missing a word?

to increase rider satisfaction and `improve` the customer experience through service efficiency and service quality.

Reply:
otherwise, looks good to me!

Reply:
thanks @matikin9 !